ImageD11/nbGui/S3DXRD/run_pbp_recon.py

```diff
@@ -0,0 +1,55 @@
+if __name__ == "__main__":
+    # horrible workaround to include id11 code path
+    import sys
+    
+    id11_code_path = sys.argv[1]
+    
+    sys.path.insert(0, id11_code_path)
+
+    import ImageD11.sinograms.point_by_point
+    import ImageD11.sinograms.dataset
+    from ImageD11 import cImageD11
+    import ast
+
+    dsfile = sys.argv[2]
+    hkltol = float(sys.argv[3])
+    fpks = float(sys.argv[4])
+    dstol = float(sys.argv[5])
+    etacut = float(sys.argv[6])
+    ifrac = float(sys.argv[7])
+    costol = float(sys.argv[8])
+    y0 = float(sys.argv[9])
+    symmetry = sys.argv[10]
+    foridx = [int(x) for x in ast.literal_eval(sys.argv[11])]
+    forgen = [int(x) for x in ast.literal_eval(sys.argv[12])]
+    uniqcut = float(sys.argv[13])
+    minpkint = int(sys.argv[14])
```

Comment:
For lima segmenter we read the parameters from the dataset (there is a group), which is already the first arg. With several pbp runs (different phases, parameters etc), we could just create a group for each one that contains the parameters at attrs and the pbpfile and maps and members / links.

Reply:
I completely agree - if you don't mind, I think this should become a wider mini-project (saving indexing parameters to the dataset object) which we could make an issue for? and in the meantime merge this PR?